winsup/cygwin/pinfo.cc

```diff
@@ -1268,7 +1268,8 @@ proc_waiter (void *arg)
       if (!ReadFile (vchild.rd_proc_pipe, &buf, 1, &nb, NULL)
 	  && GetLastError () != ERROR_BROKEN_PIPE)
 	{
-	  system_printf ("error on read of child wait pipe %p, %E", vchild.rd_proc_pipe);
+	  if (GetLastError() != ERROR_OPERATION_ABORTED)
```

Comment:
I don't particularly like calling `GetLastError` twice in a row like that (really, 3 times since apparently `%E` in system_printf does so also), but...